src/whenVisible.README.md

```diff
@@ -0,0 +1,15 @@
+# `whenIdle()`
```

Comment:
should this be for `whenVisible()` maybe?

Reply:
Oops! (thanks 😄 )